contract/rollup.config.mjs

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file rollup configuration to bundle core-eval script
+ *
+ * Supports developing core-eval script, permit as a module:
+ *   - import { E } from '@endo/far'
+ *     We can strip this declaration during bundling
+ *     since the core-eval scope includes exports of @endo/far
+ *   - `bundleID = ...` is replaced using updated/cached bundle hash
+ *   - `main` export is appended as script completion value
+ *   - `permit` export is emitted as JSON
+ */
+// @ts-check
+import {
+  coreEvalGlobals,
+  moduleToScript,
+  configureBundleID,
+  emitPermit,
+} from './tools/rollup-plugin-core-eval.js';
+import { permit as postalServicePermit } from './src/postal-service.proposal.js';
+import { permit as swapPermit } from './src/swaparoo.proposal.js';
+import { permit as sellPermit } from './src/sell-concert-tickets.proposal.js';
+import { permit as boardAuxPermit } from './src/platform-goals/board-aux.core.js';
+
+/**
+ * @param {*} opts
+ * @returns {import('rollup').RollupOptions}
+ */
+const config1 = ({
+  name,
+  coreEntry = `./src/${name}.proposal.js`,
+  contractEntry = `./src/${name}.contract.js`,
```

Comment:
😍  module type suffixes

Reply:
I'd prefer `.core.js` to `.proposal.js` but didn't manage to take on the churn.

Or maybe it should be `.deploy.js`?

Reply:
I like "deploy" as it's a _deployment script_, yeah?